sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -233,6 +233,12 @@ export interface DatasetsPendingUploadOptionalParams extends OperationOptions {
 // @public
 export type DatasetType = "uri_file" | "uri_folder";
 
+// @public
+export interface DatasetUploadOptionalOptions {
+    connectionName?: string;
+    filePattern?: RegExp;
```

Comment:
Are there no doc strings for these two values?

Reply:
I'm assuming the filePattern only applies to when uploadFolder is used? This should be documented.

Reply:
oh I see you have it documented below in the .ts file